src/NumericalAlgorithms/LinearOperators/Divergence.cpp

```diff
@@ -43,20 +36,13 @@ void divergence(
                           DerivativeFrame>& inverse_jacobian) {
   set_number_of_grid_points(div_input, mesh.number_of_grid_points());
 
-  // We have to copy into a Variables because we don't currently have partial
-  // derivative functions for anything other than Variables.
-  using VectorTag = VectorTag<Dim, DerivativeFrame>;
-  Variables<tmpl::list<VectorTag>> vars{get<0>(input).size()};
-  get<VectorTag>(vars) = input;
-  const auto logical_derivs =
-      logical_partial_derivatives<tmpl::list<VectorTag>>(vars, mesh);
+  const auto logical_deriv = logical_partial_derivative(input, mesh);
```

Comment:
Did you profile this? the copy might actually be faster because then the dgemm is on contiguous memory... This implementation I think also does 3x the memory allocations compared to the copy one.

Reply:
I added a quick benchmark and it looks like the old implementation is indeed a bit faster (maybe 1.3x), so I left it unchanged. I added the benchmark to the divergence commit.

Reply:
Thanks!